JavaScripit/index.js

```diff
@@ -18,7 +18,74 @@ var skillsList = skillsSection.querySelector('ul');
 for (var i = 0; i < skills.length; i++) {
     var skill = document.createElement("li");
     skill.innerText = skills[i];
-    skillsList.appendChild(skill);
+    skillsList.appendChild(skill); 
+}
+// DOM Selection
+const messageForm = document.forms.leave_message;
+// Event Listener
+messageForm.addEventListener('submit', function (event) {
+    event.preventDefault(); // Prevents the default refreshing behavior of the "submit" event
+
+    // Retrieve form field values
+    const usersName = event.target.usersName.value;
+    const usersEmail = event.target.usersEmail.value;
+    const usersMessage = event.target.usersMessage.value;
+
+    // Log form field values
+    console.log('Name:', usersName);
+    console.log('Email:', usersEmail);
+    console.log('Message:', usersMessage);
+    messageForm.reset();
+});
+
+// User data
+const userData = [
+    { usersName: 'Elliot', usersEmail: 'elliot@example.com', usersMessage: 'wrote:hello world' },
+    { usersName: 'Tom', usersEmail: 'grace@example.com', usersMessage: 'wrote:what am I doing here' },
+    { usersName: 'Lori', usersEmail: 'oliver@example.com', usersMessage: 'wrote:good work!' }
+];
+// Select the #messages section by id
+const messageSection = document.querySelector('#messages');
+
+// Query the messageSection to find the <ul> element
+if (messageSection) {
+    const messageList = messageSection.querySelector('ul');
+
+    // Create and append new message elements
+    userData.forEach(user => {
```

Comment:
Instead of populating the page with fake data, we want to use your code below line 56 _inside_ the event handler at line 38, so that when the form is filled out, the page's content is updated.

Reply:
Didn't get clearly the question , thanks got it now.